content/zh/docs/reference/kubernetes-api/common-parameters/common-parameters.md

```diff
@@ -0,0 +1,191 @@
+-->
+服务器保证在使用 continue 时返回的对象将与发出单一列表调用相同，没有限制——也就是说，在发出第一个请求后，创建、修改或删除的对象将不包含在任何后续的继续请求中。 
+<!--
+This is sometimes referred to as a consistent snapshot, and ensures that a client that is using limit to receive smaller chunks of a very large result can ensure they see all possible objects. If objects are updated during a chunked list the version of the object that was present at the time the first list result was calculated is returned.
+<hr>
+-->
+这有时被称为一致快照，并确保使用 limit 接收非常大的结果的较小块的客户端可以确保他们看到所有可能的对象。如果对象在分块列表中更新，则返回计算第一个列表结果时出现的对象版本。
+<hr>
+
+## namespace {#namespace}
+<!--
+object name and auth scope, such as for teams and projects
+<hr>
+-->
+
+对象名称和身份验证范围，例如用于团队和项目。
+<hr>
+
+## pretty {#pretty}
+<!--
+If 'true', then the output is pretty printed.
+<hr>
+-->
+
+如果 'true' ，那么输出是规范的打印。
+
+<hr>
+
+## propagationPolicy {#propagationPolicy}
+<!--
+Whether and how garbage collection will be performed. Either this field or OrphanDependents may be set, but not both. The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy. Acceptable values are: 'Orphan' - orphan the dependents; 'Background' - allow the garbage collector to delete the dependents in the background; 'Foreground' - a cascading policy that deletes all dependents in the foreground.
+<hr>
+-->
+该字段决定是否以及如何执行垃圾收集。可以设置此字段或 OrphanDependents，但不能同时设置。默认策略由元数据中的现有终结器设置决定。终结器和特定于资源的默认策略。可接受的值是:“孤儿”—受抚养人的孤儿; 'Background'—允许垃圾回收器删除后台的依赖;“前景”一个级联策略，删除前景中的所有依赖项。
+<hr>
+
+## resourceVersion {#resourceVersion}
+<!--
+resourceVersion sets a constraint on what resource versions a request may be served from. See https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions for details.
+
+Defaults to unset
+<hr>
+-->
+resourceVersion 对请求的资源版本设置了一个约束。详情请参见 https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions。
+
+默认设置
+<hr>
+
+## resourceVersionMatch {#resourceVersionMatch}
+<!--
+resourceVersionMatch determines how resourceVersion is applied to list calls. It is highly recommended that resourceVersionMatch be set for list calls where resourceVersion is set See https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions for details.
+
+Defaults to unset
+<hr>
+-->
+resourceVersionMatch 字段决定如何将 resourceVersion 应用于列表调用。强烈建议对设置了 resourceVersion 的列表调用设置 resourceVersion 匹配，具体请参见 https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions。
+
+默认设置
+
+<hr>
+
+## timeoutSeconds {#timeoutSeconds}
+<!--
+Timeout for the list/watch call. This limits the duration of the call, regardless of any activity or inactivity.
+<hr>
+-->
+列表/观察呼叫超时。这限制了调用的持续时间，无论是否有活动。
+<hr>
+
+## watch {#watch}
+<!--
+Watch for changes to the described resources and return them as a stream of add, update, and remove notifications. Specify resourceVersion.
+<hr>
+-->
+监视对所述资源的更改，并将其作为添加、更新和删除通知流返回。指定 resourceVersion。
```

Comment:
```suggestion
监视对所述资源的更改，并将其这类变更以添加、更新和删除通知流的形式返回。指定 resourceVersion。
```

Reply:
Thank you for your suggestions and have been modified according to the suggestions